src/server/routes/map.js

```diff
@@ -2,30 +2,28 @@ import express from 'express';
 
 const router = express.Router();
 
-router.get('/getmarkers', (req, res) => {	
-
-	let markersData = [
-		{
-			id: 'red selo',
-	        lat: 37.563398,
-	        lng: 126.9907941,
-	        title: 'KRASNOYE SELO CIRCLE',
-	        description: 'circle',
-	        address: 'circle',        
-	        type: 0        
-		},
-		{
-			id: 'alaska',
-	        lat: 37.565398,
-	        lng: 126.9907941,
-	        title: 'ALASKA',
-	        description: 'alaska',
-	        address: 'alaska',        
-	        type: 0   
-		}
-	];
-
-	res.json(markersData);	
+router.get('/getmarkers', (req, res) => {
+  let markersData = [
```

Comment:
Please use "const" instead of "let" unless you reassign the value: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.y0fz6clii